docs/specs/connecting.md

```diff
@@ -0,0 +1,115 @@
+# Connecting Devices to Cloud
```

Comment:
Breadcrumbs - [**UDMI**](../../) / [**Docs**](../) / [**Specs**](./) / [Connecting](#)

Reply:
Added, also updated the tool to check for breadcrumbs (limited form)